source/agora/utils/Log.d

```diff
@@ -115,3 +115,57 @@ public class AgoraLayout : Appender.Layout
         }
     }
 }
+
+// Log Level Definition for C++ code
+enum LogTrace = Level.Trace;
+enum LogInfo = Level.Info;
+enum LogWarn = Level.Warn;
+enum LogError = Level.Error;
+enum LogFatal = Level.Fatal;
+
+// Add "SCP" logger
+public template AddSCPLogger ()
+{
+    import ocean.util.log.Logger;
+    private Logger scplog;
+    static this ()
+    {
+        scplog = Log.lookup("SCP");
+    }
+}
+mixin AddSCPLogger!();
+
+// Logging function called from C++
+extern (C++) void DLogging(const char *logger, int level, const char *msg)
+{
+    import std.stdio;
+    import std.string;
+    
+    switch(level) {
```

Comment:
Style nitpicks:
- Braces on their own line
- space between `switch` and parenthesis

But that `switch` is actually not necessary. Look it [Logger.info](https://github.com/bpfkorea/ocean/blob/c053e665a127ec9b31cab00ff8dfc3354a1558e0/src/ocean/util/log/Logger.d#L504-L507) definition.